CHANGELOG.md

```diff
@@ -9,6 +9,10 @@
 
 ##### Enhancements
 
+* Make `trailing_comma` rule autocorrectable.
```

Comment:
requires two trailing spaces as described in CONTRIBUTING.md: https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes